spec/sample-data/valid/control-statement-04.json

```diff
@@ -0,0 +1,17 @@
+{
+  "subject_entity": {
+    "entity_properties": {
+      "company_number": "09876543",
+      "jurisdiction_code": "gb"
+    },
+    "entity_type": "company"
+  },
+  "control_level": "unknown",
+  "no_owners_reasons": [
+    "super-secure-person-with-significant-control"
```

Comment:
In `control-statement-03.json` you have the user-readable description rather than the enum value. Is that deliberate? We should talk about which one is preferable.